app/experimenter/nimbus-ui/package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Heads up: I pulled down and `yarn`'d and it generated an updated `yarn.lock` file

Reply:
Thanks for calling this out, I had a conflict with a `yarn.lock` from `main` and thought I fixed it by removing it and running `yarn` again, but I just deleted them and ran it again and also get an updated file. o.O Maybe that'll fix my CI problems... Repushing now.